lib/Runtime/Types/JavascriptEnumerator.h

```diff
@@ -40,10 +39,6 @@ namespace Js {
         // seeing this comment please just consolidate the changes.
         virtual Var MoveAndGetNext(PropertyId& propertyId, PropertyAttributes* attributes = nullptr) = 0;
```

Comment:
Can you update comment, especially on the relationship between return value & returned propertyId?